tools/build_config/build.ps1

```diff
@@ -37,10 +37,11 @@ param (
   [switch][Alias("h")]$help,
 
   # The version of Visual Studio to build with. Other Windows compilers are
-  # not supported by this script. {2015, 2017, 2019} [default: 2017]
+  # not supported by this script. {2015, 2017, 2019}
+  # [default: 0 (use latest installed version)]
   [int][ValidateSet(2015, 2017, 2019)]
   [Alias("VS")]
-  $vs_version = 2017,
+  $vs_version = 0,
```

Comment:
As you have mentioned 0 in the "default" comment it needs to be added to the ValidateSet list, otherwise change the comment to just say "default: use latest installed version" (which may not be one of the valid set....)